testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -295,7 +295,7 @@ List jobConfigs = [
         cronValue: cronOffHoursBusinessWeekdayTwiceMonthlyOdd,
         githubUserReviewers: [],
         githubTeamReviewers: [],
-        emails: ['teaching-and-learning@edx.opsgenie.net'],
+        emails: ['community-engineering@edx.org'],
```

Comment:
Lets do our Ops Genie email, I think that'd be better. `community-engineering@edx.opsgenie.net`